CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(Northstar CXX ASM_MASM)
+
+# Language specs
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_C_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_VS_PLATFORM_TOOLSET v143)
+
+# This determines the real binary root directory
+set(NS_BINARY_DIR ${CMAKE_BINARY_DIR}/game)
```

Comment:
This is good.

Reply:
wouldnt this need to be a cache entry to be user settable?
e.g.
`set(NS_BINARY_DIR "${CMAKE_BINARY_DIR}/game" CACHE STRING "Northstar binary path")`